### PR TITLE
[stable/phpMyAdmin] Add nodeSelector, tolerations and affinity for phpMyAdmin

### PR DESCRIPTION
**Pull request by antoine-rehm**
**What this PR does / why we need it**: In order to make pod assignment works

**Special notes for your reviewer**: N/A

_Relevant maintainers:_ @prydonius @tompizmor @sameersbn @carrodher @juan131

### CODE REVIEW

File: stable/phpmyadmin/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: phpmyadmin
-version: 1.0.0
+version: 1.0.1
```
Comment (cpanato) [inline stable/phpmyadmin/Chart.yaml]: New features plese bump to `1.1.0`

File: stable/phpmyadmin/README.md [hunk 2]
```diff
@@ -82,10 +85,10 @@ $ helm install --name my-release -f values.yaml stable/phpmyadmin
 
 ## Upgrading
 
-### To 1.0.0
+### To 1.1.0
 
 Backwards compatibility is not guaranteed unless you modify the labels used on the chart's deployments.
-Use the workaround below to upgrade from versions previous to `1.0.0`. The following example assumes that the release name is `phpmyadmin`:
+Use the workaround below to upgrade from versions previous to `1.1.0`. The following example assumes that the release name is `phpmyadmin`:
```
Comment (sameersbn) [inline stable/phpmyadmin/README.md]: @antoine-rehm  is this change intended?

Comment (antoine-rehm) [reply stable/phpmyadmin/README.md]: no sorry i'm going to correct that

File: stable/phpmyadmin/README.md [hunk 3]
```diff
@@ -82,7 +85,7 @@ $ helm install --name my-release -f values.yaml stable/phpmyadmin
 
 ## Upgrading
 
-### To 1.0.0
+### To 1.1.0
```
Comment (juan131) [inline stable/phpmyadmin/README.md]: Please do not bump the version here. The workaround shown below is to update from **0.x.x to 1.0.0**. Once you're on **1.0.0** you don't need to do it to upgrade to **1.1.0**

**Review by juan131 — CHANGES_REQUESTED** [review]

**Pull request merged**